.github/workflows/rust.yml

```diff
@@ -87,3 +87,21 @@ jobs:
           override: true
       - name: Run unit tests
         run: mise run rust:test
+
+  upload_coverage_report:
+    name: Run Rust test coverage
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
```

Comment:
Toss names on these steps for grepability.

Reply:
moved these elsewhere